bpf/lib/common.h

```diff
@@ -220,13 +221,21 @@ struct lb4_service {
 	__u16 port;
 	__u16 count;
 	__u16 rev_nat_index;
+	__u16 weight;
 } __attribute__((packed));
 
 struct lb4_reverse_nat {
 	__be32 address;
 	__u16 port;
 } __attribute__((packed));
 
+/* Must match lbmap.go maxSeq */
+#define LB_RR_MAX_SEQ 31
+struct lb_sequence {
+	__u16 count;
+	__u16 idx[LB_RR_MAX_SEQ];
```

Comment:
`maxSeq` needs to be `var` if you want to do that. That why you would only need to change the `CILIUM_LB_RR_MAX_SEQ` in a single place and the changes would be applied on both files.